packages/leaders-program/src/components/card/index.vue

```diff
@@ -113,6 +113,10 @@ export default {
       type: Boolean,
       default: false,
     },
+    featured: {
```

Comment:
Change this (and throughout)  to `featuredProductLabel` so it's clearer what the property is used for